MonstieWash/Assets/Scripts/Player/PlayerHealth.cs

```diff
@@ -163,5 +185,52 @@ IEnumerator StartInvincibility()
         }
 
         m_isInvincible = false;
-    }    
+    }
+
+    /// <summary>
+    /// Applies a slow and knockback away from the centre of the monster, knockback and slow reduce over time.
+    /// </summary>
+    /// <returns></returns>
+    IEnumerator ApplyKnockbackEffects()
+    {
+        //Initialise time tracking and set hand move speed to starting slow strength.
+        var timeRunning = 0f;
+        m_hand.HandSpeed = Mathf.Max(m_originalMoveSpeed - speedCurve.Evaluate(0 / slowDuration) * slowStrength, 0);
+
+        //Get Hands location
+        var handToV2 = new Vector2(m_hand.transform.position.x, m_hand.transform.position.y);
+        //Get the centre of the screen.
+        Vector2 centre;
+        if (knockBackCentralPoint != null)
+        { centre = new Vector2(knockBackCentralPoint.position.x, knockBackCentralPoint.position.y) - handToV2; }
+        else
+        { centre = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width * 0.5f, Screen.height * 0.5f, 0)); };
```

Comment:
Worth noting that in this game (0, 0) is always the centre of the screen. 

```suggestion
        { centre = Vector2.zero; };
```